tests/hazmat/primitives/test_padding.py

```diff
@@ -172,9 +179,9 @@ def __str__(self):
 
         str(mybytes())
         padder = padding.ANSIX923(128).padder()
-        padder.update(mybytes(b"abc"))
+        data = padder.update(mybytes(b"abc")) + padder.finalize()
         unpadder = padding.ANSIX923(128).unpadder()
-        unpadder.update(mybytes(padder.finalize()))
+        unpadder.update(mybytes(data))
```

Comment:
It looks like a similar change had to happen back in da45641e46 for the PKCS7 padder.